### DISCUSSION TITLE
Smaller tab character in code editor

### DISCUSSION THREAD

**Discussion by maikeldaloo** [community/community#5871] [Mobile]
Hello,

The GitHub app is really nice and polished, I'd love to see the size of the tab character reduced - they're quite large at the moment. At the very least, provide an option for people to customise. That would be awesome. 

Thanks,
Maikel

**Comment by brianlovin** [comment]
Thanks for the note @maikeldaloo — I've added your +1 to our tracking issue for making the tab size customizable in the app's Code Settings 😊
